app/src/test/java/com/github/sdp/ratemyepfl/database/ReviewRepositoryTest.kt

```diff
@@ -0,0 +1,81 @@
+package com.github.sdp.ratemyepfl.database
+
+import com.github.sdp.ratemyepfl.database.ReviewRepository.Companion.toReview
+import com.github.sdp.ratemyepfl.model.review.Review
+import com.github.sdp.ratemyepfl.model.review.ReviewRating
+import com.google.firebase.firestore.DocumentSnapshot
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito
+import org.mockito.junit.MockitoJUnitRunner
+import java.time.LocalDate
+
+@RunWith(MockitoJUnitRunner::class)
+class ReviewRepositoryTest {

```

Comment:
Why are the test commented ?

Reply:
Sorry, these tests for failing so I commented them during the merge. I fixed it now.